frontend/app/services/checkout/identitystrategy/Strategy.scala

```diff
@@ -7,12 +7,11 @@ import play.api.mvc.{RequestHeader, Result}
 import services.AuthenticationService.authenticatedIdUserProvider
 import services.{IdentityApi, IdentityService}
 
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
```

Comment:
Is there a reason we chose to use Scala's execution context [over Play's](https://www.playframework.com/documentation/2.5.x/ThreadPools#Using-the-default-thread-pool)?

Reply:
that's the import of the class definition, not the execution context implicit at runtime